google/cloud/pubsublite/BUILD.bazel

```diff
@@ -26,6 +26,7 @@ cc_library(
     deps = [
         "//:common",
         "//:grpc_utils",
+        "//:pubsub",
```

Comment:
It seems strange for "pubsub light" to depend on "pubsub heavy".  Perhaps there is some refactoring that should happen here.

Reply:
#8859